Kitodo-API/src/main/java/org/kitodo/api/dataformat/View.java

```diff
@@ -20,6 +20,20 @@
  * {@code MediaUnit} as a whole.
  */
 public class View {
+
+    /**
+     * Creates a new view with a media unit.
+     * 
+     * @param mediaUnit
+     *            media unit to set in the view
+     * @return a new view with a media unit
+     */
+    public static View of(MediaUnit mediaUnit) {
```

Comment:
Please give a more descriptive name

Reply:
This is a common pattern, as you find it in `Optional<T> Optional.of(T value)`, `Pair<L, R> Pair.of(L left, R right)`, `Stream<T> Stream.of(T t)` or `LocalDate.of(int year, int month, int dayOfMonth)`. If you don’t like it, please suggest a better one.